### ISSUE TITLE
Add a Advanced Interface

### ISSUE THREAD

**Issue by boss89** [Edivad99/ExtraStorage#5]
Basically just like the advanced exporters and importers, could you add a advanced Interface, so it has more import and export slots, would be awesome, thanks.

**Comment by Andr0oS** [comment]
Would also be neat to have it run faster by default, or work more like the AE2 interface and just suck up all the items immediately - maybe make the recipe include a stack upgrade?

**Comment by boss89** [comment]
> Would also be neat to have it run faster by default, or work more like the AE2 interface and just suck up all the items immediately - maybe make the recipe include a stack upgrade?

Agree, just noticed the normal interface stack upgrades dont do anything

**Comment by NuclearTeddyBear530** [comment]
If a return to inventory is not possible at least have an option to wait for the craft to return to inventory so you wouldn't have to use comparators to lock the crafter. Adding the additional redstone is just not possible in some cases. We would still use importers to bring the item back into inventory but have the crafter wait until the item count increases from the previous count before sending the next set
